src/main/java/teammates/storage/api/FeedbackResponsesDb.java

```diff
@@ -543,13 +543,30 @@ public FeedbackResponse getFeedbackResponseEntityOptimized (FeedbackResponseAttr
     }
     
     /**
+     * Updates the feedback response identified by {@code newAttributes.getId()} and 
+     *   changes the {@code updatedAt} timestamp to be the time of update.
+     * For the remaining parameters, the existing value is preserved 
+     *   if the parameter is null (due to 'keep existing' policy).<br> 
+     * Preconditions: <br>
+     * * {@code newAttributes.getId()} is non-null and correspond to an existing feedback response.
+     * @throws EntityDoesNotExistException 
+     * @throws InvalidParametersException 
+     */
+    public void updateFeedbackResponse(FeedbackResponseAttributes newAttributes)
+        throws InvalidParametersException, EntityDoesNotExistException {
```

Comment:
It should be 8 spaces. Let's correct it in the new code even if it is inconsistent with the old code. That way it will get corrected over time.